src/stats_tracker.cpp

```diff
@@ -66,6 +66,54 @@ int event_multiset::total( const std::string &field, const cata::event::data_typ
     return total;
 }
 
+int event_multiset::minimum(const std::string& field) const
+{
+    return minimum(field, {});
+}
+
+int event_multiset::minimum(const std::string& field, const cata::event::data_type& criteria) const
```

Comment:
Are you using this `criteria` feature?  It doesn't look like it.  If not, I suggest remove it and leave that to `event_transformation`s.  The `criteria` overloads for `count` and `total` are (IIRC) only there for historical reasons.

Reply:
Just in the tests. Cargo-culting. I'll remove them.